perun/postprocess/regression_analysis/tools.py

```diff
@@ -170,4 +173,24 @@ def r_square_safe(sse, sst):
     try:
         return 1 - sse / sst
     except ZeroDivisionError:
-        return _R_SQUARE_DEFAULT
+        return R_SQUARE_DEFAULT
+
+
+def linspace_safe(start, end, steps):
```

Comment:
This name is hard to say naturally. Can we enhance the name to something more verbose? split_space_safely? Or something similar?